.github/AUTORELEASE.md

```diff
@@ -0,0 +1,40 @@
+# Autorelease action
+
+## Overview
+Auto-release workflows automate the process of creating software releases in response to specific triggers like merging a pull request or pushing to a certain branch. This automation helps streamline the development process, reduce human error, and ensure consistent release practices.
```

Comment:
It would be better to add an indent here after this line and before the next paragraph